modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectDetails/ConfigurationContainer.tsx

```diff
@@ -135,12 +119,39 @@ export function ConfigurationContainer({
 					}}
 					onToggle={() =>
 						setValues({
-							enableObjectEntryDraft: !values.enableObjectEntryDraft,
+							enableObjectEntryHistory: !values.enableObjectEntryHistory,
 						})
 					}
-					toggled={values.enableObjectEntryDraft}
+					toggled={values.enableObjectEntryHistory}
 				/>
-			)}
+			</ClayForm.Group>
+
+			<ClayForm.Group>
+				{Liferay.FeatureFlags['LPS-181663'] && (
```

Comment:
The form group should only be created if the feature flag is enabled, no? Otherwise I think we are going to have an empty, unnecessary element in the dom if the FF is not enabled.

Reply:
Makes sense, @cgmonte, sorry for missing that! I'll adjust it now.